src/Components/LoadingState/Loading.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
This file is technically getting coverage from another file's test, but it could use its own test like [here](https://github.com/RedHatInsights/subscription-inventory-ui/blob/main/src/components/emptyState/__tests__/Processing.test.tsx).